test/performance/regression/README.md

```diff
@@ -0,0 +1,50 @@
+# Kubestellar performance regression testing
+This is a simple performance test that aims to identify major regressions. 
+
+The setup is a single bindingpolicy and a single deployment. The test adds the deployment to the WDS and measures the amount of time it takes to create the WDS object, the Binding object, the ManifestWork object, and the deployment object on the WEC. The starting time is just prior to adding the WDS object and the end time is just after the deployment object is observed on the WEC. The other timings are obtained from the objects creation timestamp. The test intentionally looks at seconds rather than milliseconds since we expect this test to run on VMs and and anything below seconds is likely just noise. In addition, the test uses creation timestamps which have seconds granuality.
```

Comment:
The description of what the test does fails to mention creating the BindingPolicy. Please insert that in its proper place in the sequence. It would also be good if the PR's opening comment included the BindingPolicy creation at its proper point in the sequence.

Reply:
Good point.  Updated.